pkg/corerp/datamodel/recipe_types.go

```diff
@@ -21,6 +21,9 @@ type RecipeConfigProperties struct {
 	// Configuration for Terraform Recipes. Controls how Terraform plans and applies templates as part of Recipe deployment.
 	Terraform TerraformConfigProperties `json:"terraform,omitempty"`
 
+	// Configuration for Bicep Recipes. Controls how Bicep plans and applies templates as part of Recipe deployment.
+	Bicep BicepConfigProperties `json:"bicep,omitempty"`
```

Comment:
I think the docs need to start with the variable name.

Reply:
updated it